src/components/Modal/Translate.vue

```diff
@@ -105,7 +127,7 @@ export default {
 					set.add(item.from)
 					result.push({
 						id: item.from,
-						label: item.fromLabel,
+						label: !this.$isMobile ? item.fromLabel : t('text', 'Translate from ' + item.fromLabel),
```

Comment:
```suggestion
						label: !this.$isMobile ? item.fromLabel : t('text', 'Translate from {language}', { language: item.fromLabel }),
```

This works better for translations into languages which don't have the language prepended to the end.